manifest.json

```
@@ -1,6 +1,6 @@
 {
   "name": "InsTab",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Don't Update the version number ri8 now.
```suggestion
  "version": "0.0.0",
```

---

okay. Any other issues in my code